README.md

```diff
@@ -24,6 +24,23 @@ The easiest way to run wireguard-ui is using the container image. To test it, ru
 
 When running in production, we recommend using the latest release as opposed to `latest`.
 
+### Configuration
+
+You can configure wireguard-ui using commandline flags or environment variables.
```

Comment:
@suom1 so you want this changed to
```suggestion
You can configure wg-ui using commandline flags or environment variables.
```
?
If so, feel free to apply the change. But I don't really see the reason. The README also calls it "wireguard-ui" in the "Running" section.